tests/spark-it/src/test/scala/org/apache/celeborn/tests/spark/CelebornHashCheckDiskSuite.scala

```diff
@@ -91,9 +91,9 @@ class CelebornHashCheckDiskSuite extends SparkTestBase {
         worker.storageManager.workingDirWriters.values().asScala.foreach { t =>
           assert(t.size() === 0)
         }
-        // after shuffle key expired, diskInfo.actualUsableSpace will equal capacity=1000
+        // after shuffle key expired, diskInfo.actualUsableSpace will equal capacity=0
```

Comment:
Shuffle keys are expired means that there is more space to use. I think this UT looks confusing.

Reply:
I think u mean the `diskInfo.totalSpace` is not empty, cause the usableSpace has to considering about the disk reserve size. 
Maybe the UT here means the totalSpace, the usableSpace and the totalSpace int the original code in this UT  will both be configuredUsableSpace.
But whatever, the UT here has nothing to do with this PR.
Is it fine that I change this condition into `diskInfo.totalSpace === 1000` ?

Reply:
> Shuffle keys are expired means that there is more space to use. I think this UT looks confusing.

I think u mean the diskInfo.totalSpace is not empty, cause the usableSpace has to considering about the disk reserve size.
Maybe the UT here means the totalSpace, the usableSpace and the totalSpace int the original code in this UT will both be configuredUsableSpace.
But whatever, the UT here has nothing to do with this PR.
Is it fine that I change this condition into diskInfo.totalSpace === 1000 ?

Reply:
done and plz help review again😁